src/text-fragments.js

```diff
@@ -37,6 +37,24 @@ import * as utils from './text-fragment-utils.js';
         parsedFragmentDirectives,
     );
     const createdMarks = processedFragmentDirectives['text'];
+
+    const targetTextStyle = utils.getTargetTextStyle();
+    const markStyle = {
+      backgroundColor: targetTextStyle && targetTextStyle.backgroundColor ?
+                        (targetTextStyle.backgroundColor.includes('!important') ?
+                          targetTextStyle.backgroundColor :
+                          targetTextStyle.backgroundColor + ' !important') :
+                        'rgb(233,210,253) !important',
+      color: targetTextStyle && targetTextStyle.color ?
+              (targetTextStyle.color.includes('!important') ?
+                targetTextStyle.color :
+                targetTextStyle.color + ' !important') :
+              null
+    };
+    for (const createdMark of createdMarks) {
+      utils.setMarkStyle(createdMark[0], markStyle);
```

Comment:
This needs a double loop--we want to apply to `createdMark[i]` for each `i` in `createdMark`. Or more succinctly I think we can do:

```
for (const mark of createdMarks.flat()) {
  utils.setMarkStyle(mark, markStyle);
}
```

(The reason this is a list of lists is that sometimes we need to use more than one `<mark>` element to wrap a given range. Each element of the outer list is a range, and each element of the inner list is a mark containing part/all of that range.)